### PR TITLE
WIP: bump kubernetes to 1.19.0-rc.2

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/cmd/openshift-apiserver/openshiftapiserver/config.go [hunk 1]
```diff
@@ -141,14 +139,16 @@ func NewOpenshiftAPIConfig(config *openshiftcontrolplanev1.OpenShiftAPIServerCon
...
 		&genericConfig.Config,
-		genericConfig.Config.LoopbackClientConfig,
-		informers.kubernetesInformers,
-		genericapiserveroptions.NewProcessInfo("openshift-apiserver", "openshift-apiserver"),
-		&genericapiserveroptions.WebhookOptions{
-			AuthInfoResolverWrapper: authInfoResolverWrapper,
-			// the openshift-apiserver runs on cluster as a normal pod, accessed by a service, so it should always have access to the service network
-			ServiceResolver: aggregatorapiserver.NewClusterIPServiceResolver(informers.kubernetesInformers.Core().V1().Services().Lister()),
-		},
+		/*
```
Comment (tnozicka) [inline pkg/cmd/openshift-apiserver/openshiftapiserver/config.go:142]: dynamic audit got dropped https://github.com/kubernetes/kubernetes/commit/ed4e6f10265ae32b1c2c0b254a4d2c20590cfadd#diff-515f037b063ce70c486bf4316e2f0789L294

**Comment by p0lyn0mial** [comment]
closing in favour of https://github.com/openshift/openshift-apiserver/pull/132
/close

**Pull request closed without merging**